src/ui/app.js

```diff
@@ -317,26 +317,26 @@ class ArchiveWebApp extends ReplayWebApp
         <div class="message is-small">
           <div class="message-body">
             <div class="buttons">
-              <button class="button is-small no-pad-mobile" title="Create New..." @click="${() => this.showNew = "show"}">
+              <button class="button is-small no-pad-mobile" title="New Archiving Session" @click="${() => this.showNew = "show"}">
                 <span class="icon">
                   <fa-icon .svg=${fasPlus}></fa-icon>
                 </span>
-                <span class="is-hidden-mobile">Create New...</span>
+                <span class="is-hidden-mobile">New Archiving Session</span>
               </button>
-              <button class="button is-small no-pad-mobile" title="Import Archive..." @click="${() => this.showImport = true}">
+              <button class="button is-small no-pad-mobile" title="Import File" @click="${() => this.showImport = true}">
                 <span class="icon">
                   <fa-icon .svg=${fasUpload}></fa-icon>
                 </span>
-                <span class="is-hidden-mobile">Import Archive...</span>
+                <span class="is-hidden-mobile">Import File</span>
               </button>
-              <button class="button is-small no-pad-mobile" title="Start Recording..." ?disabled="${!this.colls}" @click="${this.onShowStart}">
+              <button class="button is-small no-pad-mobile" title="Start Archiving" ?disabled="${!this.colls}" @click="${this.onShowStart}">
                 <span class="icon">
                   <fa-icon size="1.0em" aria-hidden="true" .svg="${wrRec}"></fa-icon>
                 </span>
-                <span class="is-hidden-mobile">Start Recording...</span>
+                <span class="is-hidden-mobile">Start Archiving</span>
               </button>
               <div class="rightbar">
-                <div class="infomsg is-hidden-mobile">The ArchiveWeb.page ${IS_APP ? "App" : "Extension"} allows you to create web archives directly in your browser!</div>
+                <div class="infomsg is-hidden-mobile">The ArchiveWeb.page ${IS_APP ? "App" : "Extension"} allows you to archive webpages directly in your browser!</div>
```

Comment:
Do we have a style guide take on web pages vs. webpages yet? I see them used interchangeably elsewhere and I think either's fine but we might want to make sure we're consistent on that between apps/docs.

Reply:
We don't really have a _style guide_ yet!!  There's the documentation one but I've punted doing this to when we eventually have a single component library and therefore a storybook site with the documentation style guide acting as bonus info for documentation only.

I'm a "webpage" guy I think, but super willing to go with whatever we decide on this one... Might continue to punt this to later and make note of it in my Design System Plans obsidian note?